### DISCUSSION TITLE
how to add multiple files in dropzone on drag

### DISCUSSION THREAD

**Discussion by Yashsharma1911** [react-dropzone/react-dropzone#1162] [Q&A]
As I use maxFiles this help me to take multiple files but only when I open file dialog but whenever I try to add multiple file on drag it always resets to one file

**Comment by rolandjitsu** [comment]
@Yashsharma1911 I'm not sure I understand what you're trying to achieve. Selecting multiple files or dragging multiple files should be possible. What do you mean by `resets to one file`?

**Comment by rolandjitsu** [comment]
No reply here, so closing.
